packages/cli/src/apollo/cat-facts.example.json

```diff
@@ -0,0 +1,189 @@
+{
```

Comment:
I've deliberately checked this in so that we have some kind of test data for template gen. I'm not wild about it, it's a bit of a knee-jerk decision really